include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -36,15 +39,28 @@
 #include "dlaf/matrix/panel.h"
 #include "dlaf/matrix/tile.h"
 #include "dlaf/matrix/views.h"
+#include "dlaf/schedulers.h"
 #include "dlaf/sender/keep_future.h"
 #include "dlaf/sender/traits.h"
+#include "dlaf/types.h"
+#include "dlaf/util_math.h"
 #include "dlaf/util_matrix.h"
 
 #include "dlaf/eigensolver/reduction_to_band/api.h"
 #include "dlaf/factorization/qr.h"
 
 namespace dlaf::eigensolver::internal {
 
+// Given a vector of vectors, reduce all vectors in the first one using sum operation
+template <class T>
+void reduceColumnVectors(std::vector<common::internal::vector<T>>& battery) {
+  for (size_t i = 1; i < battery.size(); ++i) {
```

Comment:
Here and everywhere else in this PR:
```suggestion
  for (std::size_t i = 1; i < battery.size(); ++i) {
```
Also, include `cstddef`.